fedn/fedn/clients/reducer/restservice.py

```diff
@@ -958,10 +958,10 @@ def checksum():
 
             file_path = os.path.join(UPLOAD_FOLDER, name)
             print("trying to get {}".format(file_path))
-            from fedn.utils.checksum import md5
+            from fedn.utils.checksum import sha
```

Comment:
I know there's a lot of this in fedn, but perhaps we can start to move imports to the beginning of the file?

Reply:
that way it will also be handled by isort

Reply:
fixed